cfgov/locale/es/LC_MESSAGES/django.po

```diff
@@ -188,25 +188,6 @@ msgstr "Sí"
 msgid "optional"
 msgstr "opcional"
 
-#: v1/_includes/ask/ask-search-bar.html
```

Comment:
I think most (maybe all) of these translations are still needed, they're just no longer scoped to the ask-search-bar template.

Reply:
So would I just be deleting line 191, or updating it to the `ask-search.html` template?

Reply:
Oh wait, I see this is an autogenerated file and that's just a comment. Running https://github.com/cfpb/cfgov-refresh/blob/master/docs/translation.md#2-run-the-makemessages-management-command-to-add-the-string-to-the-portable-object-file I see a whole bunch changed. I'll revert this change and leave it for another PR.